### DISCUSSION TITLE
Server side user settings

### DISCUSSION THREAD

**Discussion by IsJustaGhost** [jellyfin/jellyfin#12875] [General]
Back when I was making addons for Elder Scrolls Online, we had the option of giving the user the option to save setting for the addon as "Account wide" or "per Character".
Account wide means the settings would be the same on every character they log in on, as long as they are from the same account.

Well, that was just a thought about the following.

The user setting could have a toggle for saving like that. Some user setting can be saved client side or server side. 
Server side user setting would remain even when the user is forced to clear site data from the browser, or log in on another machine or browser.
Server side settings, to me, would really be the only reason to set the display mode to auto. It would allow it to change from tv or mobile to desktop depending on what device one logs in on.
client side, one would have to set display mode to the desired setting on whatever device they are on. Really, as is, what's the point in Auto?
If I log in on a PC, I'm logged in on a PC. If I log in on a phone or tablet, then I'm on mobile. It can't really change, can it? I mean, the user settings "have!" to be set on the device, for the device. On a phone I could set it to Desktop or TV , but why? I'm on a mobile device so I'd set it to mobile.
I'm just complaining about having only used Jellyfin on a PC and have had to re set the user settings every time I log in.
"every time I log in".. If I don't have to log in, the settings are still in local site data. If I have to log in, I most likely have had to clear site data.

I'd actually prefer to have the option to store all my user settings server side.
It seams like I must set my desired display settings every time I use Jellyfin.
Screensaver, backdrops, display mode.

**Comment by crobibero** [comment]
This is already available, and is a per-client decision.

**Reply by IsJustaGhost** [reply]
Exists? In what way?

**Reply by crobibero** [reply]
There exists an endpoint that allows clients to store client preferences. As an example it is used in the web client to store home screen preferences.

**Reply by IsJustaGhost** [reply]
My experience with that, as stated, was having to re set those user setting on every login. They do not carry across. As, if I have to log in, I had to clear the browsers site data. 
Besides that. If I log in as local host, then I still have to change it again if I log in with IP.
I should clarify, I'm talking about on the same client.

**Reply by IsJustaGhost** [reply]
I'm not sure but, is this what is being discussed here?  [https://features.jellyfin.org/posts/315/mysql-server-back-end](url) If only partly.

**Comment by gnattu** [comment]
> Screensaver, backdrops, display mode.

If those are want you meant then unfortunately these are designed to be client side specific and not to be linked with an account because it does not make any sense to make it to. Users don't want the same screensaver settings, backdrops and display mode on different device and make them linked to a user account will create more problems than it solves.

**Reply by gnattu** [reply]
I'm fully aware that for your use case you want to it being saved somewhere more reliable than the browser local data, but link it user account is definitely not the correct way to do this because this would break the use case for other users with a lot of devices.

**Reply by IsJustaGhost** [reply]
I appreciate the input. What about a user's default settings. Something something that one would fallback on if/when they log onto a new device or otherwise log in and the local data was lost?

Instead of the applications defaults. 
On first log in. settings would fall back on local settings > user default > application default
Move to next if not available.

**Comment by HaoSs07** [comment]
To bad this request was just ignored, I find the lack of any general settings for users server side very disturbing.

**Comment by IsJustaGhost** [comment]
What requst
I am only on a tablet I don't see the full email
________________________________

**Comment by IsJustaGhost** [comment]
Had a stroke. Hostage now. Can't leave.
________________________________
